tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -56,11 +62,13 @@ export default async function (tree: Tree, schema: MigrateConvergedPkgGeneratorS
   };
 }
 
-function runBatchMigration(tree: Tree, userLog: UserLog) {
-  const projects = getProjects(tree);
+function runBatchMigration(tree: Tree, userLog: UserLog, projectNames?: string[]) {
+  const projects = Array.from(getProjects(tree)).filter(([projectName]) => {
+    return projectNames ? projectNames.includes(projectName) : true;
+  });
```

Comment:
very minor nitpick, but I feel it improves clarity a bit:
```suggestion
  const projects = Array.from(getProjects(tree))
  const projectsToRun = !projectNames ? projects : project.filter(([projectName]) =>
    projectNames.includes(projectName)
  );
```

If we go with this, remember to also update references to `projects` or alternatively it could use `projects -> allProjects` and `projectsToRun -> projects` naming.

Reply:
thanks for suggestion. this is bit subjective opinion and every POV can be different. Also using negatives first (especially in ternaries) lowers the readability and cognitive load from my experience.

I created override of devkit `getProjects` that enables filtering